tests/test_physical_channel_property.py

```diff
@@ -0,0 +1,65 @@
+"Contains a collection of pytest tests that validates the physical channel properties."
+import os
+
+from nidaqmx.constants import TerminalConfiguration
+from nidaqmx.tests.test_read_write import TestDAQmxIOBase
+
+
+class TestPhysicalChannelProperty(TestDAQmxIOBase):
+    """Contains a collection of pytest tests that validates the physical channel properties."""
+
+    def test_bool_property(self, any_x_series_device):
+        """Test for validating boolean attributes in physical channel."""
+        phys_chans = any_x_series_device.di_lines
+        assert phys_chans[0].di_change_detect_supported
+
+    def test_byte_property(self, any_x_series_device):
+        """Test for validating byte attributes in physical channel."""
+        phys_chans = any_x_series_device.ai_physical_chans
+
+        # Generate path to a virtual TEDS file.
+        teds_file_path = os.path.join(os.path.dirname(__file__), "teds", "Voltage.ted")
```

Comment:
FYI, I'm moving this to `test/test_assets/teds/Voltage.ted` and adding a `test_assets_directory` fixture. 

https://github.com/ni/nidaqmx-python/pull/245